lib/node-package-manager.ts

```diff
@@ -53,6 +54,12 @@ export class NodePackageManager implements INodePackageManager {
 	public view(packageName: string, propertyName: string): IFuture<any> {
 		return this.loadAndExecute("view", [[packageName, propertyName], [false]]);
 	}
+
+	public executeNpmCommand(npmCommandName: string, currentWorkingDirectory: string): IFuture<any> {
+		return (() => {
```

Comment:
`return this.$childProcess.exec(npmCommandName, { cwd: currentWorkingDirectory })`